### ISSUE TITLE
Find out which mouse buttons are pressed

### ISSUE THREAD

**Issue by 00v1** [jquery/jquery#2168]
I need a way to check which mouse button is pressed.
I tried to record mousedown and mouseup events, but when I release two mouse buttons off screen, only one mouseup event is fired.
Also I can not cheat around it by using mouseleave events.

http://jsfiddle.net/3gfa05gd/3/

**Comment by motapc97** [comment]
I believe [StackOverflow](http://stackoverflow.com/) is the right place to do that kind of question.

But, can I ask you what exactly are you trying to accomplish? I don't see any problem in your JSFiddle.

**Comment by 00v1** [comment]
Ok, let me rephrase the problem:

JQuery does not correctly fire mouseup events when multiple buttons are released off screen.

Expected workflow:
Press two mouse buttons, move mouse off screen, release two mouse buttons, get two mouseup events

Current workflow:
Press two mouse buttons, move mouse off screen, release two mouse buttons, only get one mouseup event

**Comment by motapc97** [comment]
Oh, I see what you mean now. Thanks. I've tested, and it also happens with 3 mouse buttons pressed and then released: only 1 mouseup event.

But I think that's the way browsers handle those events.

Tested without jQuery: [http://jsfiddle.net/3gfa05gd/4/](http://jsfiddle.net/3gfa05gd/4/)

**Comment by 00v1** [comment]
So it's a browser bug then?

**Comment by motapc97** [comment]
I don't know. But happened to me in Chrome, Firefox and Opera. In IE11, no mouseup event happens (weird...).

**Comment by 00v1** [comment]
Ah well, if javascript still can't get such basic things as mouse api right I'll just use flash instead. Thanks for your help anyway.
